flink-libraries/flink-table/src/test/scala/org/apache/flink/table/ExpressionReductionTest.scala

```diff
@@ -155,15 +155,15 @@ class ExpressionReductionTest extends TableTestBase {
       "DataSetCalc",
       batchTableNode(0),
       term("select",
-        "13 AS _c0",
+        "CAST(13) AS _c0",
```

Comment:
I will have a look at it again. In general, the only real solution is finally fix FLINK-5177.